README.md

```diff
@@ -65,9 +65,9 @@ generate_parameter_library(
 
 add_executable(minimal_node src/turtlesim.cpp)
 target_link_libraries(minimal_node PRIVATE
-  rclcpp::rclcpp
   turtlesim_parameters
 )
+ament_target_dependencies(minimal_node rclcpp)
```

Comment:
I do not believe it is "more correct" to use ament_target_depenencies because it does not support INTERFACE/PUBLIC/PRIVATE linking and when you require the use of scoped linking using ament_target_dependencies prevents you from also using target_link_libraries. @ChrisThrasher

Reply:
Is there some problem you are experiencing when using target_link_libraries that ament_target_dependencies solves?

Reply:
I'm not aware of any advantages `ament_target_dependencies` gives us in this context.

Reply:
In this concrete case, no, but on the other PR yes when having multiple `target_link_libraries` and “PRIVATE” linking.

I find the other method just much easier to read.